x/dex/types/keys.go

```diff
@@ -45,16 +45,20 @@ func PairPrefix(priceDenom string, assetDenom string) []byte {
 }
 
 func OrderBookPrefix(long bool, contractAddr string, priceDenom string, assetDenom string) []byte {
+	return append(
+		OrderBookContractPrefix(long, contractAddr),
+		PairPrefix(priceDenom, assetDenom)...,
+	)
```

Comment:
Do we need to worry about where this is called?

Reply:
this is just a refactor so the caller wouldn't see a difference

Reply:
bbut in the old method it appends the `append(prefix` wheras this doesnt?